package.json

```diff
@@ -11,31 +11,41 @@
   },
   "dependencies": {
     "@emotion/react": "^11.11.4",
+    "@emotion/style": "^0.8.0",
     "@emotion/styled": "^11.11.5",
     "@fontsource/roboto": "^5.0.13",
     "@mui/icons-material": "^5.15.18",
-    "@mui/material": "^5.15.18",
+    "@mui/material": "^5.15.19",
     "@mui/styled-engine-sc": "^6.0.0-alpha.18",
+    "@mui/styles": "^5.15.19",
+    "@mui/x-charts": "^7.6.2",
+    "@mui/x-data-grid": "^7.6.1",
+    "@mui/x-data-grid-generator": "^7.6.1",
+    "@mui/x-data-grid-pro": "^7.6.1",
+    "@types/axios": "^0.14.0",
     "axios": "^1.7.2",
-    "react": "^18.2.0",
+    "formik": "^2.4.6",
+    "jwt-decode": "^4.0.0",
+    "react": "^18.3.1",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.51.5",
+    "react-loader-spinner": "^6.1.6",
```

Comment:
We can select one from this page and include it in our project without depending upon an external package.
https://cssloaders.github.io/